src/Util.hs

```diff
@@ -33,8 +33,9 @@ import           Data.Word                    as X
 import           GHC.Generics                 as X (Generic)
 import           Numeric.Natural              as X (Natural)
 
-import           Control.Monad.Except         as X (ExceptT (..), MonadError (..), runExceptT)
+import           Control.Monad.Except         as X (MonadError (..))
 import           Control.Monad.Identity       as X
+import           Control.Monad.Trans.Except   as X (ExceptT (..), runExceptT)
```

Comment:
This is due to https://github.com/haskell/mtl/issues/117 of course. Since we don't know when the fix will be published, maybe it makes sense to release this change anyway.

Reply:
Yes, since there is the import of `MonadFix` which warrants a re-release, we can have this as well...